index.html

```diff
@@ -0,0 +1,328 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="mag-style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <title>Smashing Mag Grayscale</title>
+</head>
+<body>
+
+    <!--         HEADER          -->
+    <header class="dark">
+        <nav>
+            <!-- <div class="logo">Logo</div> -->
+            <p>Header Section</p>
+        </nav>
```

Comment:
Let's add the navigation element with `div`

##### Navigation bar

* [ ] It uses `div` and positions elements as at the source page (content is not important).

Reply:
implemented by @silverside3000